tests/Zend/Log/Writer/StreamTest.php

```diff
@@ -55,6 +55,8 @@ public function testConstructorThrowsWhenResourceIsNotStream()
         } catch (Exception $e) {
             $this->assertTrue($e instanceof Zend_Log_Exception);
             $this->assertRegExp('/not a stream/i', $e->getMessage());
+        } catch (TypeError $e) {
+            $this->assertRegExp('/must be of t/i', $e->getMessage());
```

Comment:
I am not sure about the adjustments here. Were these `TypeError`, `ValueError` triggered as regular errors in PHP <8 or were they simply non-errors before and now PHP 8 is more strict? If that's the case, is it okay that PHP8 will throw such errors instead of the php <8 shrug-and-proceed behavior?

Reply:
Usually where I added Error catches the behaviour in <8 was to emit a warning or fail silently.

Reply:
Okay I guess it is fine in case of this component - when something is wrong with internals of this stream thing, it should throw